ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -280,6 +280,52 @@ def test_clear_foreign_config_fail(self, mock_validate_job_queue,
             exception.DracOperationError, drac_raid.clear_foreign_config,
             self.node, 'RAID.Integrated.1-1')
 
+    @mock.patch.object(drac_job, 'validate_job_queue', spec_set=True,
+                       autospec=True)
+    def test_change_physical_disk_state(self,
+                                        mock_validate_job_queue,
+                                        mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+        drac_raid.change_physical_disk_state(
+            self.node,
+            mode="JBOD")
+
+        mock_client.change_physical_disk_state.assert_called_once_with(
+            "JBOD")
+
```

Comment:
Probably want to also validate that validate_job_queue was called once.
It would be nice to mock up a return value from mock_client.change_physical_disk_state().  You could then take the return value from drac_raid.change_physical_disk_state() and compare it against the expected return value here.

Reply:
Yes done. 
1. check `validate_job_queue` called once.
2. compare expected and actual of `change_state_result(`drac_raid.change_physical_disk_state()`) `